### DISCUSSION TITLE
关于对图层本身施加 蒙板路径 导出不生效的问题

### DISCUSSION THREAD

**Discussion by Xiaolianmao** [Tencent/libpag#1702] [Q&A] [answered]
环境:
macOS Ventura13.4 | AE  23.4.0(Build 53) | PAGViewer 4.3.3.94(beta)

问题:
在制作过程中对图层本身施加蒙板路径插件导出不生效

我试过的解决方法:
有问题的图层转为BMP(无效,转为bmp后还是一样不生效)
使用轨道遮罩代替(根据效果应用,有部分可以这么制作)

文件⬇️
[问题文件.zip](https://github.com/Tencent/libpag/files/12484281/default.zip)

**Comment by newstarchen** [comment, marked as answer]
原因：蒙版路径有两个关键帧，这两个关键帧的路径形状（称形状A、形状B）是两个不同构的图形（顶点个数不同），这种情况pag在插值时有BUG。
解决：最终的解决方法需要pag sdk修改兼容，这里我先提供给你一个从素材上避开的方法：在第一个关键帧之后紧邻一帧插入一个关键帧 + 在最后一个关键帧之前紧邻的一帧插入一个关键帧。（只增加关键帧即可，关键帧内容用系统默认生成的，不用人工调整）。

**Reply by Xiaolianmao** [reply]
感谢
